get_hashsets.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const AppConstants = require("./app-constants").init();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const pkg = require("./package.json");
+
+const HIBP_AUTH = `Bearer ${AppConstants.HIBP_API_TOKEN}`;
+const USER_AGENT = `${pkg.name}@${pkg.version}`;
+const BREACH_HASHSET_DIR = "breach_hashsets";
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.includes("Email addresses")) {
+    console.log("Active, verified breach with email addresses: " + breach.Name);
+    const url = AppConstants.HIBP_API_ROOT + "/enterprisesubscriber/hashset/" + breach.Name;
+    console.log("url: " + url);
+
+    const headers = {
+      "User-Agent": USER_AGENT,
+      "Authorization": HIBP_AUTH
+    };
+    const hashsetRequestObject = {
+      url: url,
+      headers: headers
+    };
+
+    request(hashsetRequestObject).pipe(fs.createWriteStream(path.join("breach_hashsets", `${breach.Name}.zip`)));
```

Comment:
nit: this `"breach_hashsets"` in the path should now use the `BREACH_HASHSET_DIR` const for improved greatness.

Which also makes me question whether that should be added to the app-constants.js file, in the case where we may need to access that directory in other places in the future when we want to _consume_ those files. But I'm also fine with leaving that tweak until the time we need to specify the path twice in two different files.